extensions/amp-form/0.1/test/test-form-dirtiness.js

```diff
@@ -98,6 +105,15 @@ describes.realWin('form-dirtiness', {}, env => {
       changeInput(textField, 'changed');
       expect(form).to.have.class(DIRTINESS_INDICATOR_CLASS);
     });
+
```

Comment:
These tests look the same to me, so maybe you only need one test for the `hasFieldBeenSubmitted_` (however you need to test it from public API). It's okay to leave it as it is now, but if you would like to have duplicated tests for checkbox, radio, options and so on, then I guess it makes sense to make it a standalone test.

Reply:
Yea I figured to leave them because they are checking two different elements after all (the same applies to checkbox et al)

Reply:
Okay sure, I don't feel too strong about this.